templates/_macros/vf_basic-section.jinja

```diff
@@ -279,7 +279,7 @@
 {% endmacro %}
 
 {% macro basic_section_title(title={}) %}
-  <h2>
+  <h2{% if "id" in title and title.get("id", "") | trim | length > 0 %} id="{{ title.get("id", "") | trim }}"{% endif %}>
```

Comment:
I think it would be better if the whole section have the id and not just the title